Sources/App/Settings/SettingsViewController.swift

```diff
@@ -102,11 +102,19 @@ class SettingsViewController: HAFormViewController {
         }
 
         if contentSections.contains(.general) {
+            #if targetEnvironment(macCatalyst)
+            form +++ Section()
```

Comment:
a few things here:

you can do a `let section = Section()` and append to it then `form +++ section` if you wanna avoid duplicated code here

you can also use the section's visibility modifier to hide on catalyst, e.g.

```
Section {
  $0.hidden = .isCatalyst
}
```

you may also find `Current.isCatalyst` easier than target defines -- less weird code paths